src/main/java/pl/nogacz/checkers/board/Board.java

```diff
@@ -404,4 +476,129 @@ public static boolean isThisSameColor(Coordinates coordinates, PawnColor color)
     public static PawnClass getPawn(Coordinates coordinates) {
         return board.get(coordinates);
     }
+
+    class EditMenu extends JFrame implements  ActionListener {
+

```

Comment:
Yeni bir dosyaya taşı. İhtiyaç duyulan değerler için accessor metotları tanımla.